html/idiom315.html

```diff
@@ -1,7 +1,33 @@
-<h1>NYI: Idiom # 315: Memoization</h1>
+<h1>Idiom # 315: Memoization</h1>
 
 <p>See <a class="shylink" target="_blank" href="https://www.programming-idioms.org/idiom/315/memoization">programming-idioms.org</a>:</p>
 
 <h2>Code</h2>
 
+<p>Let's look at two code examples that solve the same problem:
+  How many ways are there to climb an 'n' stairs, taking 1 or 2 steps at the time?
+</p>
+
+<p>Both versions use memoization to avoid unnecessary calculations, but they implement
+  it in slightly different ways. You can examine about memoization effect in fuzion language
+  from here <a href= "https://fuzion-lang.dev/docs/base/memoize+%28no+arguments%29/">link </a>
```

Comment:
```suggestion
  <a href= "https://fuzion-lang.dev/docs/base/memoize+%28no+arguments%29/">here</a>
```
the word "here" could also be the link